simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -0,0 +1,202 @@
+package io.prometheus.client.dropwizard;
+
+import com.codahale.metrics.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedMap;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+/**
+ * Collect dropwizard metrics from a MetricRegistry and export these metrics
+ * as prometheus metrics.
+ *
+ */
+public class DropwizardExports extends io.prometheus.client.Collector {
+    private MetricRegistry registry;
+    private static final Logger LOGGER = Logger.getLogger(DropwizardExports.class.getName());
+    private static String GENERIC_HELP_MESSAGE = "Generated from dropwizard import";
+    private TimeUnit timeUnit;
+
+    /**
+     * @param registry a metric registry to export in prometheus
+     * @param timeUnit the TimeUnit to use for rate metrics (default to use milliseconds)
+     */
+    DropwizardExports(MetricRegistry registry, TimeUnit timeUnit) {
+        this.registry = registry;
+        this.timeUnit = timeUnit;
+    }
+
+    DropwizardExports(MetricRegistry registry) {
+        this(registry, TimeUnit.MILLISECONDS);
+    }
+
+    /**
+     * Export counter as prometheus counter.
+     */
+    List<MetricFamilySamples> fromCounter(String name, Counter counter) {
+        MetricFamilySamples.Sample sample = new MetricFamilySamples.Sample(name, new ArrayList<String>(), new ArrayList<String>(),
+                new Long(counter.getCount()).doubleValue());
+        return Arrays.asList(new MetricFamilySamples(name, Type.COUNTER, GENERIC_HELP_MESSAGE, Arrays.asList(sample)));
+    }
+
+    /**
+     * Export gauge as a prometheus gauge
+     */
+    List<MetricFamilySamples> fromGauge(String name, Gauge gauge) {
+        Object obj = gauge.getValue();
+        Double value;
+        if (obj instanceof Integer) {
+            value = Double.valueOf(((Integer) obj).doubleValue());
+        } else if (obj instanceof Double) {
+            value = (Double) obj;
+        } else if (obj instanceof Float) {
+            value = Double.valueOf(((Float) obj).doubleValue());
+        } else if (obj instanceof Long) {
+            value = Double.valueOf(((Long) obj).doubleValue());
+        } else {
+            LOGGER.warning(String.format("Invalid type for Gauge %s: %s", name,
```

Comment:
I can see this getting very spammy with strings, it's probably best as a debug log.